docs/packaging.md

```diff
@@ -247,12 +258,10 @@ Note that the configuration is still in YAML format, but reduced in scope.
 ```sh
 apt install aziot-edge
 
-iotedge init
+iotedge config
```

Comment:
All this should be moved to the iotedge repo too. We should only talk about `aziotctl config ...` here.